server/models/Profile.js

```diff
@@ -3,26 +3,29 @@ const mongoose = require("mongoose");
 const availabilitySchema = new mongoose.Schema({
   date: {
     type: String,
-    required: true
+    required: true,
   },
   timeStart: {
     type: String,
-    required: true
+    required: true,
   },
   timeEnd: {
     type: String,
-    required: true
-  }
+    required: true,
+  },
 });
 
 const profileSchema = new mongoose.Schema({
   firstName: {
     type: String,
-    required: true
   },
   lastName: {
     type: String,
-    required: true
+  },
+  sitter: {
+    type: Boolean,
+    required: true,
+    default: false,
```

Comment:
Is this the only expected change here?

Reply:
@moffatethan slack is down, and I wanted to update you on this.